.github/workflows/deploy-docs.yaml

```diff
@@ -1,18 +1,27 @@
-name: Deploy docs to GH pages
+name: Deploy docs
 
 on:
   push:
-    branches: ["main"]
+    branches:
+      - "main"
+    tags:
+      - "*.*.*"
+      - "v*.*.*"
```

Comment:
Let's keep only the one we're using.
The other one is not supposed to trigger anything.

Reply:
Fixed.